doc/source/release/1.13.0-notes.rst

```diff
@@ -37,7 +30,9 @@ New features
 
 `scipy.interpolate` improvements
```

Comment:
```suggestion
### scipy.integrate improvements
- The `terminal` attribute of event functions passed to `scipy.integrate.solve_ivp` may now be a positive integer: integration stops after `terminal` events have occurred.

`scipy.interpolate` improvements
```

Reply:
Yeah, I've already got this one locally. I guess we'll see how things compare when I push up in a bit.